src/components/image/index.test.tsx

```diff
@@ -99,35 +99,21 @@ describe("Image component", () => {
       expect(SolidFns.getUrl).toHaveBeenCalledWith(mockThing, mockProperty);
     });
 
-    it("When getUrl returns null, with no errorComponent, should throw an error", () => {
+    it("When getUrl returns null, with an errorComponent, should render the error", () => {
```

Comment:
Isn't the point of `it("...")` meant to be that the code becomes readable like an English sentence, so this should be:
```suggestion
    it("should render the error when getUrl returns null, with an errorComponent, ", () => {
```
(I'm unsure (without context again) what "**_with an errorComponent_**" means here, so my suggestion still reads a bit weirdly, but I'm sure you get the point!).